_dashboards/visualize/tsvb.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: TSVB
+parent: Building data visualizations
+nav_order: 45
+---
+
+# TSVB
+
+The Time-Series Visual Builder (TSVB) is a powerful data visualization tool in OpenSearch Dashboards that allows you to create detailed time-series visualizations. One of its key features is the ability to add annotations or markers at specific time points based on index data. This feature is particularly useful for making connections between multiple indexes and building visualizations that display data over time, such as flight status, delays by type, and more. TSVB currently supports the following visualization types: Area, Line, Metric, Gauge, Markdown, and Data Table.
+
+## Creating TSVB visualizations from multiple data sources
+Introduced 2.14
+{: .label .label-purple }
+
+Before proceeding, ensure that the following configurations are enabled. These can be found in `config/opensearch_dasboards.yaml`:
+
+```yaml
+data_source.enabled: true
+vis_type_timeseries.enabled: true
+```
+{% include copy-curl.html %}
+
+**Step 1: Connect data sources**
+
+Once you have configured your YAML file, open OpenSearch Dashboards and follow these steps:
+
+1. From the **Home** page, select **Add sample data** and then select the **Add data** button for the **Sample web logs** dataset, as shown in the following GIF.
```

Comment:
This step should be last in the list since the data source connection has to be established before installing data (reference https://github.com/opensearch-project/project-website/pull/2837 for updated steps).